book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
```

Comment:
Kindly add an 's' to `@rental` since you called the attribute accessor `:rentals`.